catalog/spatial/csw/spatial-csw-endpoint/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/reader/TestTransactionMessageBodyReader.java

```diff
@@ -626,6 +638,20 @@ public void testConversionExceptionWhenNoConstraintInUpdate() throws IOException
         IOUtils.toInputStream(UPDATE_REQUEST_NO_CONSTRAINT_XML));
   }
 
+  @Test(expected = ForbiddenClassException.class)
+  public void testForbiddenDeserialization() throws IOException {
+    TransactionMessageBodyReader reader =
+        new TransactionMessageBodyReader(
+            mock(Converter.class), CswQueryFactoryTest.getCswMetacardType(), registry);
+    reader.readFrom(
+        CswTransactionRequest.class,
+        null,
+        null,
+        null,
+        null,
+        IOUtils.toInputStream(DYNAMIC_PROXY_SERIALIZED_XML));
```

Comment:
❗️ Although this is just unit test code, you should still close your streams wherever possible. If this were a case where you could not (and it were still just test code), you'd want to annotate this method to ignore the specific static analysis error. But there's nothing preventing this from being closed.

Reply:
I don't believe streams are closed for any of the tests in this file. Should an effort be made to annotate/close all of them?